src/pages/profiles/[publicKey]/offers.tsx

```diff
@@ -208,15 +208,15 @@ const OfferPage = (props: WalletDependantPageProps) => {
             return nft.offers
               .filter((o) => o.buyer === profilePK)
               ?.sort(byDate)
-              .map((offer) => getActivityCard(nft as Nft, offer as Offer));
+              .map((offer) => getActivityCard(nft as Nft | any, offer as Offer));
```

Comment:
Is `| any` necessary or is there a more narrowly defined type you could use?

Reply:
i have a TODO to add marketplaceProgramAddress to the sdk this is a temp fix until that update

Reply:
you can extend the Nft interface or join to together for this call.

```typescript
type IFooBar = IFoo & IBar;
```

https://stackoverflow.com/questions/49723173/merge-two-interfaces